x-pack/metricbeat/module/aws/s3/_meta/data.json

```diff
@@ -0,0 +1,39 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "agent": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "aws": {
+        "s3": {
+            "bucket": {
+                "name": "test-s3-ks",
+                "size": {
+                    "bytes": 696445
+                },
+                "storage": {
+                    "type": "StandardStorage"
+                }
+            },
+            "object": {
+                "count": 1
+            }
+        }
+    },
+    "cloud": {
+        "provider": "s3",
```

Comment:
That looks odd?

Reply:
yeah, should this be `aws` instead? https://github.com/elastic/ecs#cloud here under `cloud.provider` it has `ec2` as an example.

Reply:
We decide to remove cloud.provider for now and opened an issue in ecs to track this: https://github.com/elastic/ecs/issues/317